apps/erp/app/styles/docs.css

```
@@ -130,7 +130,7 @@
   display: flex;
   border-bottom-width: 1px;
   --tw-border-opacity: 1;
-  border-color: hsl(var(--border) / var(--tw-border-opacity));
+  border-color: color-mix(in oklab, var(--border) calc(var(--tw-border-opacity * 100%), transparent));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)